src/jsonpointer/include/sourcemeta/jsontoolkit/jsonpointer.h

```diff
@@ -31,6 +31,9 @@ namespace sourcemeta::jsontoolkit {
 /// @ingroup jsonpointer
 using Pointer = GenericPointer<JSON::String>;
 
+/// @ingroup jsonpointer
+using WeakPointer = GenericPointer<std::reference_wrapper<const std::string>>;
```

Comment:
@tony-go Can also add `#include <functional> // std::reference_wrapper`?

Reply:
Funny that it did not break on linux haha